src/components/toaster/toaster.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '@styles/variables';
+
+.viewport {
+    position: fixed;
+    inset: 0;
+    display: flex;
+    justify-content: center;
+    pointer-events: none;
+    z-index: 5;
```

Comment:
What's 5?

Reply:
Toast should over the cart. Setting '5' I reserve a few values in case there will be some objects over the cart but under the toast. What is the best practice to choose `z-index`?